SalesforceCommerceCloud/pwa-kit
# Fix: a11y - Ensure the credit card tooltip stays open after hover @W-12627113@

# Description

This PR fixes the accessibility issue where users cannot use a screen magnifier on the credit card informational tooltip since it disappears when you are not actively hovering on it.

The new behavior is that the tooltip persists after you hover it, and will disappear when you click on the page or press any button on the keyboard.

![Screenshot 2024-07-11 at 12 03 29 PM](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/88680517/c997941f-20fa-40d3-b400-dcd60fda4091)



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- (change1)

# How to Test-Drive This PR

1. `git checkout ju/a11y-fix-cc-tooltip`
2. start the retail react app
3. add an item to cart and proceed to checkout
4. once you get to the payment section, hover over the tooltip
5. observe that the tooltip persists on the page until you click somewhere or press a keyboard button

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
